pixi.toml

```diff
@@ -79,6 +79,7 @@ ace = "*"
 asio = "*"
 assimp = "*"
 boost = "*"
+cli11 = "*"
```

Comment:
Shouldn't this be placed under the "ROBOTOLOGY_USES_GZ" dependencies for clarity?

Reply:
Good point. To be honest, as it is quite a small library, I prefer to keep it in the group of generic dependencies, so that if other libraries start depend on it I do not need to remember to move it.

Reply:
Ok thanks!